config/site_settings.yml

```diff
@@ -2644,16 +2644,6 @@ uncategorized:
     client: true
     default: false
 
-  summarization_strategy:
-    client: true
-    default: ""
-    enum: "SummarizationStrategy"
-    validator: "SummarizationValidator"
-  custom_summarization_allowed_groups:
```

Comment:
I believe we want to keep these hidden so we can migrate data. We can drop them later.